src/openmmio.py

```diff
@@ -669,7 +669,7 @@ def prepare(self, pbc=False, mmopts={}, **kwargs):
                 elif self.FF.amoeba_pol == 'direct':
                     self.mmopts['polarization'] = 'direct'
             self.mmopts['rigidWater'] = self.FF.rigid_water
-            if self.FF.restrain_h== True:
+            if self.FF.constrain_h is True:
```

Comment:
I think this is a typo. I found it because it raises an `AttributeError` in examples with OpenMM engine.